include/hip/hcc_detail/hip_runtime.h

```diff
@@ -106,15 +109,16 @@ extern int HIP_TRACE_API;
 #include <hip/hcc_detail/host_defines.h>
 #include <hip/hcc_detail/math_functions.h>
 #include <hip/hcc_detail/device_functions.h>
+#if __HCC___
```

Comment:
There is an extra ```_``` at the end which breaks all texture and surface directed tests on HCC path.